deployment/docker/kafka-puller/run_app.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/bash
 
-bootstrap_server="$1"
+bootstrap_servers="$1"
 topic_name="$2"
 
 # The below variable will contain the classpath for all the jar dependencies.
 jars=$(ls extracted/*/*/*.jar | tr \\n :)
 
 while true
 do
-  java -cp build/libs/KafkaPrinter.jar:"$jars" org.opensearch.KafkaPrinter -b "$bootstrap_server" -t "$topic_name" | nc -v localhost 9210
+
+  # Verify netcat can connect to Replayer before process begins
+  while ! nc -v -w 1 localhost 9210
+  do
+    >&2 echo "Attempting to connect to Traffic Replayer..."
+    sleep 2
+  done
+  sleep 30
```

Comment:
I don't understand this code.  Is there a way that you can do dependencies in Fargate?  You can (& should) in compose definitions.

Reply:
Why is this sleep 30 seconds and the one below 7?  Why do you need any sleeps at all?

Reply:
This is ugly code and needs to be reworked in future change. Sleeps here are a "hack" to allow time for the replayer and comparator to start up